src/test/java/com.epam.rp/api/AllureSampleTest.java

```diff
@@ -4,10 +4,10 @@
 import io.qameta.allure.Feature;
 import org.junit.jupiter.api.Test;
 
-@Feature("Login")
+@Feature("SampleFeature")
 public class AllureSampleTest {
```

Comment:
Could you please clarify why this class called `AllureSampleTest`? As I see the `ApiSampleTest` will be more sutable.

Reply:
Renamed, thanks